shared/Views/Shared/Components/CourseDetails/_AboutCourse.cshtml

```diff
@@ -9,18 +9,25 @@
   var section = Model.GetHtmlForSection(CourseDetailsSections.AboutTheCourse);
 }
 
-@if (Model.PreviewMode || Model.HasSection(CourseDetailsSections.AboutTheCourse))
-{
-  <div class="govuk-!-margin-bottom-8">
-    <h2 class="govuk-heading-l" id="section-about">About the course</h2>
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-l" id="section-about">About the course</h2>
 
-    @if (section != null)
+  @if (section != null)
+  {
+    @Html.Raw(section)
+  }
+  else
```

Comment:
make this an `else if` to eliminate nesting

reject (sry, nested ifs in templates give me cramps)